lib/cubits/dish_cubit.dart

```diff
@@ -10,8 +10,9 @@ class DishCubit extends Cubit<List<Dish>> {
   DishCubit()
       : _dishService = GetIt.I.get<DishService>(),
         super([]) {
-    refresh();
+    refresh(null);
   }
 
-  Future<void> refresh() async => emit(await _dishService.fetchDishes());
+  Future<void> refresh(String? dishDay) async =>
```

Comment:
Use brackets to make `dishDay` optional (default value is `null` since `String?` is nullable).
This would allow calling `refresh();` as usual.

```suggestion
  Future<void> refresh([String? dishDay]) async =>
```